### PR TITLE
New providers added

### PR DESCRIPTION
**Pull request by chatelao**
OpenFireMap, OpenInfraMap

### CODE REVIEW

**Comment by jieter** [comment]
@chatelao thanks for your PR. Can you pllease have a look at [why the tests fail](https://travis-ci.org/leaflet-extras/leaflet-providers/builds/363492106), and fix those issues?

**Comment by chatelao** [comment]
Fix applied (formating error)

File: leaflet-providers.js [hunk 1]
```diff
@@ -132,19 +138,61 @@
...
+			}
+		},
 		OpenSeaMap: {
 			url: 'https://tiles.openseamap.org/seamark/{z}/{x}/{y}.png',
 			options: {
 				attribution: 'Map data: &copy; <a href="http://www.openseamap.org">OpenSeaMap</a> contributors'
 			}
 		},
+		OpenPtMap: {
+			maxZoom: 17,
```
Comment (jieter) [inline leaflet-providers.js]: I think this one has to go into `options`?

Comment (jieter) [reply leaflet-providers.js]: The top level object should only have keys `url`, `options` and `variants`, so the `maxZoom` should not be in `L.TileLayer.Provider.providers.OpenPtMap` but in `L.TileLayer.Provider.providers.OpenPtMap.options`.

**Comment by chatelao** [comment]
You are so nice and helpful. I will commit after fixing. Maybe you tests support checking a json schema? I could try to create one.

**Comment by chatelao** [comment]
Next try :-)

**Comment by chatelao** [comment]
Thank you for your pation with a newbie.

**Comment by jieter** [comment]
I added a bit more strictness to the tests, which can be seen [here](https://github.com/leaflet-extras/leaflet-providers/commit/226300907eae391829922ce077214a14b065e52c#diff-188d5ebe4f2ed2ebb4d8d3b86ce56137L39).

This looks good to me, I'm merging.

For next time, it's helpful to include all layer's you are adding with the PR to the description of the PR and if possible include links to their usage terms.

**Comment by jieter** [comment]
@chatelao if I may suggest some improvements:

- [OpenStreetMap.CH](http://leaflet-extras.github.io/leaflet-providers/preview/#filter=OpenStreetMap.CH) could use bounds, as it is really only defined for Switzerland.
- [OpenInfraMap.*](http://leaflet-extras.github.io/leaflet-providers/preview/#filter=OpenInfraMap) are overlays, so it would be nice if the preview would render them on a background layer. This would mean adding a regexp to [`isOverlay`](https://github.com/leaflet-extras/leaflet-providers/blob/30233ec8c27c9bd4723632f23f5879250e20b72b/preview/preview.js#L56-L67)
- Same is true for OpenRailwayMap, OpenFireMap and SafeCast.

**Comment by chatelao** [comment]
Okay, I will try this. Didn't see this part of the code (yet).

**Pull request merged**